index.html

```diff
@@ -262,6 +265,53 @@ <h2>
 did:web:example.com%3A3000
         </pre>
     </section>
+    <section>
+      <h2>
+        Key Material and Document Handling
+      </h2>
+      <p>
+        Due to the way most web servers present content, it is likely that a
+        particular did:web document will be served with a media type of
+      <pre>application/json</pre>. If a document is retrieved and it is named
+      <pre>did.json</pre> a few processing rules should apply:
+      <ul>
+        <li>
+          If an
+          <pre>@context</pre> is present at the root of the json document,
+          the document should be processed according to the JSON-LD rules.
+          If this is not possible, or if the document failes processing, the
+          document should be rejected from consideration as a did:web doc.
+        </li>
+        <li>
+          If an
+          <pre>@context</pre> is present at the root of the json document,
+          and is passes JSON-LD processing, and it contains the context
+          <pre>https://www.w3.org/ns/did/v1</pre>, it may be further processed
```

Comment:
```suggestion
          `@context` is present at the root of the JSON document,
          and it passes JSON-LD processing, and it contains the context
          `https://www.w3.org/ns/did/v1`, it may be further processed
```